SwrveCommonSDK/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'maven'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion '21.1.2'
+    buildToolsVersion '23.0.1'
```

Comment:
:+1: Be mindful of tests failing due to this though. We've had issues with some roboelectrics as a result of build and compile versions.